src/deployment/deploy.py

```diff
@@ -694,42 +689,48 @@ def set_instance_config(self) -> None:
             tenants.append(tenant)
         update_allowed_aad_tenants(config_client, tenants)
 
-    def create_eventgrid(self) -> None:
-        logger.info("creating eventgrid subscription")
-        src_resource_id = self.results["deploy"]["fuzz-storage"]["value"]
-        dst_resource_id = self.results["deploy"]["func-storage"]["value"]
-
+    def remove_eventgrid(self) -> None:
         credential = AzureCliCredential()
         client = StorageManagementClient(
             credential, subscription_id=self.get_subscription_id()
         )
-        event_subscription_info = EventSubscription(
-            destination=StorageQueueEventSubscriptionDestination(
-                resource_id=dst_resource_id, queue_name="file-changes"
-            ),
-            filter=EventSubscriptionFilter(
-                included_event_types=[
-                    "Microsoft.Storage.BlobCreated",
-                    "Microsoft.Storage.BlobDeleted",
-                ]
-            ),
-            retry_policy=RetryPolicy(
-                max_delivery_attempts=30,
-                event_time_to_live_in_minutes=1440,
-            ),
-        )
+        try:
+            storage_accounts = client.storage_accounts.list_by_resource_group(
```

Comment:
It might be useful to factor this entire `try` block out into a function that just returns the user data/`fuzz<..>` storage account.

Then this could be:
```suggestion
            src_resource_id = user_data_storage_account().id
```
(or similar)